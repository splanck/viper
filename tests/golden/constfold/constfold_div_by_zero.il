il 0.1.2
func @main() -> i64 {
entry:
  %t0 = sdiv.chk0 1, 0
  ret %t0
}
