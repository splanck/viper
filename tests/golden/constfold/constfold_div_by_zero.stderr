runtime trap: DivideByZero @ main: entry[#0]: divide by zero in sdiv.chk0
